codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpBindingProtocolGenerator.kt

```diff
@@ -4,6 +4,8 @@
  */
```

Comment:
This and package changes that follow are due to moving message marshal/unmarshal generators into a subdirectory for organization.